crates/typed-store/src/rocks/mod.rs

```diff
@@ -351,7 +352,6 @@ impl RocksDB {
         fail_point!("delete-cf-before");
         let ret = delegate_call!(self.delete_cf_opt(cf, key, writeopts));
         fail_point!("delete-cf-after");
-        #[allow(clippy::let_and_return)]
```

Comment:
You're doing the opposite of what you did in another place?

Reply:
hmm, that is odd indeed. Not sure why this isn't complaining